src/component/field.js

```diff
@@ -17,10 +17,10 @@ const amountStyles = StyleSheet.create({
   },
 });
 
-export const AmountInputField = ({ style, ...props }) => (
+export const AmountInputField = ({ fiat, style, ...props }) => (
   <HorizontalExpandingTextInput
-    style={[amountStyles.input, style]}
-    charWidth={46}
+    style={[amountStyles.input, { textAlign: fiat ? 'left' : 'right' }, style]}
```

Comment:
I wouldn't change the style in the first iteration. Important part is getting the logic right. We can revisit styling later.